youtube_dl/extractor/go90.py

```diff
@@ -32,11 +33,26 @@ def _real_extract(self, url):
             video_id, headers={
                 'Content-Type': 'application/json; charset=utf-8',
             }, data=b'{"client":"web","device_type":"pc"}')
-        title = video_data['title']
         main_video_asset = video_data['main_video_asset']
 
+        episode_number = int_or_none(video_data.get('episode_number'))
+        series = None
+        season_number = None
+        if 'Item' in video_data.get('__children', {}):
+            for key in video_data['__children']['Item']:
```

Comment:
you can iterate here using `values` method and make it in a single line without checking if `Item` is present:
```python
for metadata in video_data.get('__children', {}).get('Item', {}).values():
```